templates/api/knowledge-panels/health/ingredients/ingredient.tt.json

```diff
@@ -0,0 +1,37 @@
+{
+    "level": "info",
+    "topics": [
+        "health"
+    ],
+    "size": "small",
+    [% IF panel.ingredient_description OR panel.wikipedia_abstract %]
+    "expanded": false,
+    [% ELSE %]
+    "expanded": false,
+    [% END %]
+    "title_element": {
+        // Note: the app displays line feeds as line feeds...
```

Comment:
```suggestion
        // Note: the app displays line feeds as line feeds...
        // so we have to keep whole title expression on one line !
```

Reply:
I just added a double backtick function so that we can have things on multiple lines that are then converted to one line without linebreaks.